wiki/Beatmap_Spotlights/Seasons/2020_Summer/en.md

```diff
@@ -261,41 +261,41 @@ The following leaderboards are also available as a [Google Spreadsheet](https://
 | #13 | [yobrevelc](https://osu.ppy.sh/users/14128407) | ![][mania-platinum2] |
 | #14 | [miminc](https://osu.ppy.sh/users/8472615) | ![][mania-platinum2] |
 | #15 | [Blight-kun](https://osu.ppy.sh/users/7785754) | ![][mania-platinum1] |
-| #16 | [\[Illustrious\]](https://osu.ppy.sh/users/4082595) | ![][mania-platinum1] |
-| #17 | [Hellnote102](https://osu.ppy.sh/users/2218066) | ![][mania-platinum1] |
-| #18 | [akace100](https://osu.ppy.sh/users/9308128) | ![][mania-platinum1] |
-| #19 | [tsusai](https://osu.ppy.sh/users/7637292) | ![][mania-platinum1] |
-| #20 | [Gaymie](https://osu.ppy.sh/users/2377299) | ![][mania-platinum1] |
-| #21 | [Hidden is fun](https://osu.ppy.sh/users/10449071) | ![][mania-gold2] |
-| #22 | [Muse Dash](https://osu.ppy.sh/users/13695676) | ![][mania-gold2] |
-| #23 | [medium kek](https://osu.ppy.sh/users/11625617) | ![][mania-gold2] |
-| #24 | [Suichan203](https://osu.ppy.sh/users/9821346) | ![][mania-gold2] |
-| #25 | [Cynplytholowazy](https://osu.ppy.sh/users/3901754) | ![][mania-gold2] |
-| #26 | [GhostRevenge](https://osu.ppy.sh/users/8980266) | ![][mania-gold2] |
-| #27 | [Crystal](https://osu.ppy.sh/users/1646397) | ![][mania-gold2] |
-| #28 | [enjoyit1025](https://osu.ppy.sh/users/2206844) | ![][mania-gold2] |
-| #29 | [Raveille](https://osu.ppy.sh/users/1388767) | ![][mania-gold2] |
-| #30 | [Gato2499](https://osu.ppy.sh/users/7857559) | ![][mania-gold2] |
-| #31 | [Dilly_P](https://osu.ppy.sh/users/12907431) | ![][mania-gold2] |
-| #32 | [A M D](https://osu.ppy.sh/users/5321112) | ![][mania-gold1] |
-| #33 | [Callionet](https://osu.ppy.sh/users/3072921) | ![][mania-gold1] |
-| #34 | [cocoalatte](https://osu.ppy.sh/users/9059651) | ![][mania-gold1] |
-| #35 | [Doofyplayer](https://osu.ppy.sh/users/9828552) | ![][mania-gold1] |
-| #36 | [Joylintp](https://osu.ppy.sh/users/13579078) | ![][mania-gold1] |
-| #37 | [Sebaex](https://osu.ppy.sh/users/4686036) | ![][mania-gold1] |
+| #16 | [enjoyit1025](https://osu.ppy.sh/users/2206844) | ![][mania-platinum1] |
+| #17 | [\[Illustrious\]](https://osu.ppy.sh/users/4082595) | ![][mania-platinum1] |
+| #18 | [Sebaex](https://osu.ppy.sh/users/4686036) | ![][mania-platinum1] |
+| #19 | [Hellnote102](https://osu.ppy.sh/users/2218066) | ![][mania-platinum1] |
+| #20 | [akace100](https://osu.ppy.sh/users/9308128) | ![][mania-platinum1] |
+| #21 | [tsusai](https://osu.ppy.sh/users/7637292) | ![][mania-platinum1] |
```

Comment:
![image](https://user-images.githubusercontent.com/19843905/97749174-e4e9d380-1aee-11eb-9cee-61512937c32b.png)

Reply:
Eh, I missed this cause it doesn't really matter but fixed nevertheless.